org.gnome.Notes.json

```diff
@@ -0,0 +1,145 @@
+{
+    "app-id" : "org.gnome.Notes",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bijiben",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=host:ro",
```

Comment:
Sorry for being so stubborn about it… Doesn't `GtkFileChooserNative` use portals when available, or Notes can import entire directory, not particular files?

Reply:
Without --filesystem=host, I can't see folders to navigate into.

Reply:
I just checked the code and tested it again, that's why I re-re-re-edited the comment above.